### DISCUSSION TITLE
[Feature] LLAVA support via ollama

### DISCUSSION THREAD

**Discussion by RandomLegend** [immich-app/immich#13583] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,

i just stumbled across [hoarder](https://github.com/hoarder-app/hoarder/), it can take in images and pipe those over to ollama and then use llava to automatically tag the images.

Would something like this feasible for immich aswell? I don't have much success using the smart search on my instance even though i have a properly implemented GPU setup for it.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
What would this add over the existing CLIP search? AIUI LLaVA is basically just CLIP and an LLM glued together.
